ovc/machines.go

```diff
@@ -164,13 +170,12 @@ func (s *MachineServiceOp) List(cloudSpaceID int) (*MachineList, error) {
 }
 
 // Get individual machine
-func (s *MachineServiceOp) Get(id string) (*MachineInfo, error) {
+func (s *MachineServiceOp) Get(id int) (*MachineInfo, error) {
```

Comment:
While looking at that: there are quite a few more of these "stringly-typed" interfaces, e.g. 
```
func (s *CloudSpaceServiceOp) Get(cloudSpaceID string) (*CloudSpace, error)
func (s *DiskServiceOp) Get(diskID string) (*DiskInfo, error)
func (s *ExternalNetworkServiceOp) Get(id string) (*ExternalNetworkInfo, error)
...
```
While cleaning this up is probably out of scope for this one, I think it should be done in one go to avoid burning through major releases.